### ISSUE TITLE
NPU support on yolov8

### ISSUE THREAD

**Issue by weberwcwei** [openvinotoolkit/openvino_notebooks#2398]
**Describe the bug**
When attempting to compile the YOLOv8 model using the NPU, a RuntimeError occurs.

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[8], line 7
      5 model = core.read_model(model_path)
      6 model.reshape([1, 3, 640, 640])
----> 7 compiled_model = core.compile_model(model, "NPU")

File c:\Users\Weber\Desktop\.conda\Lib\site-packages\openvino\runtime\ie_api.py:543, in Core.compile_model(self, model, device_name, config, weights)
    538     if device_name is None:
    539         return CompiledModel(
    540             super().compile_model(model, {} if config is None else config),
    541         )
    542     return CompiledModel(
--> 543         super().compile_model(model, device_name, {} if config is None else config),
    544     )
    545 else:
    546     if device_name is None:

RuntimeError: Exception from src\inference\src\cpp\core.cpp:107:
Exception from src\inference\src\dev\plugin.cpp:53:
Exception from src\plugins\intel_npu\src\plugin\src\plugin.cpp:697:
Exception from src\plugins\intel_npu\src\plugin\src\compiled_model.cpp:62:
Exception from src\plugins\intel_npu\src\compiler\src\zero_compiler_in_driver.cpp:853:
L0 pfnCreate2 result: ZE_RESULT_ERROR_INVALID_ARGUMENT, code 0x78000004

```
**Sample code**
```
from pathlib import Path
import openvino as ov

models_dir = Path("./model_zoo")
DET_MODEL_NAME = "yolov8n"

model_path = models_dir / f"{DET_MODEL_NAME}_openvino_model/{DET_MODEL_NAME}.xml"
core = ov.Core()
model = core.read_model(model_path)
model.reshape([1, 3, 640, 640])
compiled_model = core.compile_model(model, "NPU")
```

 **Installation instructions (Please mark the checkbox)**
[X] I followed the installation guide at https://github.com/openvinotoolkit/openvino_notebooks#-installation-guide to install the notebooks. 

**Additional context**
NPU driver version: 32.0.100.2714

**Comment by brmarkus** [comment]
Can you provide details about the platform you are using, which HW you are running on?
Which operating system do you use?
Which version of OpenVINO do you use?

When using the current version of the notebook "http://localhost:8888/lab/tree/yolov8-optimization/yolov8-object-detection.ipynb" then I still see the problem of failing compilation with like `[...] from unsupported opset: opset14`, as mentioned in other issues, for which workarounds exist (see e.g. "https://github.com/openvinotoolkit/openvino_notebooks/issues/2167#issuecomment-2269827840").

**Comment by weberwcwei** [comment]
Hi @brmarkus ,

Here are the platform details:
- OS: Windows 11
- OpenVINO: 2024.5.0.dev20240911
- NPU: Meteor Lake Ultra 5

Follow your mention, I tried changing the maxpool operation from opset14 to opset8 in xml, and the model compiled successfully.

I also find a similar open issue in here (https://github.com/openvinotoolkit/openvino/issues/26510).

**Comment by brmarkus** [comment]
I think the dev-team is still working on this, please expect follow-up questions from them, like asking you to try a newer version or patches.

**Comment by avitial** [comment]
This has been fixed with OpenVINO 2024.5 release and the latest NPU driver. Closing issue, please reopen if issue persists.

**Comment by chen-conghua** [comment]
This issue is still when I work with the NPU. 
My CPU is Ultra 228V and the driver version is 32.0.100.3104
The OPENVINO version is 2024.5.0

Jupyter Notebook: https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/tensorflow-object-detection-to-openvino/tensorflow-object-detection-to-openvino.ipynb

The error report is:  
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[19], line 3
      1 core = ov.Core()
      2 openvino_ir_model = core.read_model(openvino_ir_path)
----> 3 compiled_model = core.compile_model(model=openvino_ir_model, device_name=device.value)

File c:\Users\conghua\miniconda3\envs\openvino-test\Lib\site-packages\openvino\runtime\ie_api.py:543, in Core.compile_model(self, model, device_name, config, weights)
    538     if device_name is None:
    539         return CompiledModel(
    540             super().compile_model(model, {} if config is None else config),
    541         )
    542     return CompiledModel(
--> 543         super().compile_model(model, device_name, {} if config is None else config),
    544     )
    545 else:
    546     if device_name is None:

RuntimeError: Exception from src\inference\src\cpp\core.cpp:107:
Exception from src\inference\src\dev\plugin.cpp:53:
Exception from src\plugins\intel_npu\src\plugin\src\plugin.cpp:711:
Exception from src\plugins\intel_npu\src\compiler_adapter\src\ze_graph_ext_wrappers.cpp:389:
L0 pfnCreate2 result: ZE_RESULT_ERROR_INVALID_ARGUMENT, code 0x78000004 - generic error code for invalid arguments . Cannot get Interpolate Sizes value at loc(fused<{name = "Preprocessor/ResizeToRange/cond/resize/ResizeBilinear", type = "Interpolate"}>["Preprocessor/ResizeToRange/cond/resize/ResizeBilinear"])
